nautobot/extras/models/models.py

```diff
@@ -667,23 +670,23 @@ class Notes(BaseModel, ChangeLoggedModel):
         null=True,
     )
     user_name = models.CharField(max_length=150, editable=False)
-    name = models.CharField(max_length=150)
-    slug = AutoSlugField(populate_from="name", max_length=150)
+
+    slug = AutoSlugField(populate_from="assigned_object", max_length=150)
```

Comment:
Can this follow the default slug length of 100, or do we actually need to allow 150?

Reply:
The idea I had was that some models will have really long names, like Jobs, so to allow enough, then 150. But with the truncate you added, I think it is safe to remove max_length.